pulpcore.te

```diff
@@ -135,7 +139,29 @@ optional_policy(`
 	kerberos_read_keytab(pulpcore_server_t)
 ')
 
+optional_policy(`
+	postgresql_stream_connect(pulpcore_t)
+	postgresql_stream_connect(pulpcore_server_t)
```

Comment:
In Katello (Satellite) we also support an external PostgreSQL instance. Is that also allowed with this configuration?

Reply:
Connecting over tcp has already been allowed in the policy:
pulpcore.te:corenet_tcp_connect_postgresql_port(pulpcore_t)
pulpcore.te:corenet_tcp_connect_postgresql_port(pulpcore_server_t)